cookbook-react/src/recipes/radios/zecatDifapro/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import i18next from 'i18next';
+import clsx from 'clsx';
+
+import styles from './styles.module.scss';
+import ValidIcon from './assets/ic_valid.svg';
+import InvalidIcon from './assets/ic_invalid.svg';
+
+interface Props {
+  title: string;
+  info: string;
+  extraInfo: string;
+  disabled?: boolean;
+  name: string;
+  id: string;
+  inputStyle?: 'normal' | 'custom';
+  type?: 'radio' | 'checkbox';
+  checked: boolean;
+  onChange?: (event: React.FormEvent<HTMLInputElement>) => void;
+  wrapperClassName?: string;
+  titleClassName?: string;
+  infoClassName?: string;
+  extraInfoClassName?: string;
+}
+
+function ZecatDifapro({
+  title = i18next.t('ZecatDifapro:title'),
+  info = i18next.t('ZecatDifapro:info'),
+  extraInfo = i18next.t('ZecatDifapro:extraInfo'),
+  disabled = false,
+  name,
+  id,
+  inputStyle = 'normal',
+  type = 'radio',
+  checked = false,
+  onChange,
+  wrapperClassName,
+  titleClassName,
+  infoClassName,
+  extraInfoClassName
+}: Props) {
+  if (inputStyle === 'custom') {
+    return (
+      <div
+        className={clsx(
+          styles.wrapper,
+          `${wrapperClassName ?? ''}`,
+          { [styles.default]: !disabled, [styles.disabled]: disabled, [styles.selected]: checked },
+          'row middle'
+        )}
+        onChange={onChange}
+      >
+        <div className={`${styles.firstSection} row`}>
+          {checked ? (
+            <input
```

Comment:
Since the only 2 things that change depending on the checked prop are the src and the alt, you can do the following:

```
<input
              type="image"
              src={checked ? ValidIcon : InvalidIcon}
              alt={i18next.t(`ZecatDifapro:${checked ? 'selected' : 'notSelected'}Img`) as string}
              name={name}
              id={id}
              className={styles.customIcon}
              data-testid={`${name}-zecatdifapro`}
/>
```

And you avoid repeating the rest of the props